webpack/scenes/Subscriptions/components/SubscriptionsTable/SubscriptionsTableHelpers.js

```diff
@@ -1,22 +1,33 @@
 const buildTableRow = (subscription, availableQuantities, updatedQuantity) => {
-  const availableQuantityLoaded = !!availableQuantities;
-  const availableQuantity = availableQuantityLoaded
+  const upstreamAvailableLoaded = !!availableQuantities;
+  const upstreamAvailable = upstreamAvailableLoaded
     ? availableQuantities[subscription.id]
     : null;
 
+  let maxQuantity;
+  if (upstreamAvailableLoaded && (upstreamAvailable !== undefined)) {
```

Comment:
You don't have to check if `upstreamAvailable` is defined here since its defined above